Products/zms/conf/metaobj_manager/com.zms.foundation/ZMSTable/interface0.zpt

```diff
@@ -382,7 +388,7 @@
 				function table_cell_edit(el_td) {
 					var el = $('textarea',el_td);
 					var el_name = el.attr('id');
-					table_cell_id = el_name.substring(el_name.indexOf('_')+1);
+					table_cell_id = el_name.substr(el_name.indexOf('_')+1);
```

Comment:
substr() is Deprecated: This feature is no longer recommended.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr?retiredLocale=de